### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Agur4ik4**
New Name

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
 print("Muhammad Aliyu")
 print("Monotosh Ghosh")
-print("Alex James")
\ No newline at end of file
+print("Alex James")
+print("Deniss Aleksejev")
```
Comment (crimacode) [inline main.py:3]: I think the name should be "Denis" instead of "Deniss"

File: main.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
 print("Muhammad Aliyu")
```
Comment (crimacode) [inline main.py]: Lets remove this person because he has relocated to Canada and no longer in Nigeria

File: main.py (hunk 1 above)
Comment (Agur4ik4) [reply main.py:3]: I thought as well! But my documents says it is correct

File: main.py (hunk 2 above)
Comment (Agur4ik4) [reply main.py]: The name is removed

**Review by crimacode — APPROVED** [review]

**Pull request merged**